webCommander/centralHub/views.py

```diff
@@ -36,3 +37,17 @@ def handle_summary(request, twitter_handle):
 
 def home_page(request):
     return render(request, 'home.html')
+
+def list_artist(request):
+    allartists = SpotifyArtistInfo.objects.all()
+
+    return render(request, 'spotifyFramework/spotify_summary.html',{'allartists':allartists})
+
+def get_artists(request):
+    name = request.GET.get("artist", "")
+    response = search_spotify_artist(name)
+
+    options = ''.join(
+        "<option value='{val}'>{val}, Popularity: {pop}</option>".format(val=row[0], pop=row[2]) for row in response
```

Comment:
not sure why I suggested format here, nothing wrong with it, but you can also use an f-string :)